### DISCUSSION TITLE
Adaptive Theme Colour

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#880] [Ideas]
### What feature would you like to see?

There's an extension called [Adaptive Tab Bar Colour](https://github.com/easonwong-de/Adaptive-Tab-Bar-Colour), It changes the Firefox theme to match the website's theme. The extension seems to be broken on Zen but it would be nice to have this feature built in.
Here's an example from the extension's page.

![image](https://github.com/user-attachments/assets/784fe63e-b82f-4d43-99eb-fe2014116af1)

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/224_

**Comment by enoversum** [comment]
Just adding my vote to this. While some user on Mac reports it to be working, me and someone else isn't able to enable this functionality. The add-on reports it successfully sampled a colour off a page, but Zen doesn't show it in the tabbar.
